tests/python/rest_api/test_jobs.py

```diff
@@ -574,9 +576,16 @@ def test_get_gt_job_in_org_task(
     "restore_db_per_class"
 )
 class TestGetGtJobData:
-    @pytest.mark.usefixtures("restore_db_per_function")
+
+    def _delete_gt_job(self, user, gt_job_id):
+        with make_api_client(user) as api_client:
+            (_, gt_job_del_response) = api_client.jobs_api.destroy(gt_job_id)
+            assert (
+                gt_job_del_response.status == HTTPStatus.NO_CONTENT
+            ), "newly created gt_job couldn't be deleted"
```

Comment:
```suggestion
            api_client.jobs_api.destroy(gt_job_id)
```

The call will raise an exception if the request fails.

Reply:
Oh thanks was unaware about it.